sematic/cli/run.py

```diff
@@ -53,24 +56,51 @@ def _run_example(example_path: str):
 
 @cli.command(
     "run",
-    short_help="Run a pipeline, or a packaged example",
-    context_settings=dict(
-        ignore_unknown_options=True,
-        allow_extra_args=True,
-    ),
+    short_help="Run a pipeline, or a packaged example.",
+    context_settings=dict(ignore_unknown_options=True, allow_extra_args=True),
 )
 @click.argument("script_path", type=click.STRING)
+@click.option(
+    "-b",
+    "--build",
+    help=(
+        "Build a container image that will be used to execute the pipeline, "
+        "using the configured build plugin. "
+        "If not specified, the pipeline will be ran locally in the current environment."
+    ),
+    default=False,
+    is_flag=True,
+)
 @click.pass_context
-def run(ctx, script_path: str):
-    if not server_is_running():
-        click.echo("Sematic is not started, issue `sematic start` first.")
-        return
+def run(ctx, script_path: str, build: bool):
+
+    # if not server_is_running():
+    #     click.echo("Sematic is not started, issue `sematic start` first.", err=True)
+    #     sys.exit(1)
```

Comment:
This is a blocker that will be have to be solved in order to submit pipelines natively to non-local Servers.